code/modules/tgui_panel/tgui_panel.dm

```diff
@@ -1,3 +1,6 @@
+/// Each client notifies on protected playback, so this prevents spamming admins.
+GLOBAL_VAR_INIT(admins_warned, FALSE)
```

Comment:
this 1000% needs a more specific name if it's going to be a global and not a static var